sensorimotor/sensorimotor/one_d_universe.py

```diff
@@ -19,7 +19,8 @@
 # http://numenta.org/licenses/
 # ----------------------------------------------------------------------
 
-from nupic.data.pattern_machine import PatternMachine, ConsecutivePatternMachine
+from nupic.data.generators.pattern_machine import PatternMachine,\
```

Comment:
Wrap the two imports in parens for implicit line continuation rather than using backslash